newsletter-march-2023.rst

```diff
@@ -31,17 +31,16 @@ News and updates
   If you find an unofficial and unmaintained project,
   :doc:`please see our policy on unofficial and unmaintained projects <readthedocs:unofficial-projects>`.
 - 🐞 Custom 404s with Sphinx DirHTML builder `are now supported <https://github.com/readthedocs/sphinx-notfound-page/issues/215>`__.
-- 🐞 :doc:`Pull Request <readthedocs:guides/pull-requests>` builds should point to previews, not build pages. The fix will be deployed on March 7.
-- 🔒️ Vulnerability fixed: Pull requests preview builds `GHSA-h4cf-8gv8-4chf <https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-h4cf-8gv8-4chf>`__
-- 🔒️ Vulnerability fixed: Access to files from any project possible with crafted path traversal, see: `GHSA-5w8m-r7jm-mhp9 <https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-5w8m-r7jm-mhp9>`__.
+- 🔒️ Vulnerability fixed: `Serving content from pull requests previews on main docs domains <https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-h4cf-8gv8-4chf>`__.
+- 🔒️ Vulnerability fixed: `Path traversal: access to files from any project <https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-5w8m-r7jm-mhp9>`__.
 
 You can always see the latest changes to our platforms in our :doc:`Read the Docs Changelog <readthedocs:changelog>`.
 
 .. _rclone: https://rclone.org/
 
 
-Upcoming features
------------------
+Upcoming
+--------
```

Comment:
Renamed the section so it can also hold "upcoming (other things)" such as bug fixes.

Reply:
I put in another comment why I think the bug fixes are not too relevant 😄

Reply:
Yea, I gave that feedback in the original newsletter PR as well. I'm not sure why we're including them in the newsletter.